app/views/users/mfa_selection/index.html.erb

```diff
@@ -24,6 +24,8 @@
   <%= f.submit t('forms.buttons.continue'), class: 'margin-bottom-1' %>
 <% end %>
 
-<%= render 'shared/cancel', link: @after_setup_path %>
+<%= render PageFooterComponent.new do %>
+  <%= link_to t('mfa.skip'), @after_setup_path, method: :get, class: 'usa-button usa-button--unstyled' %>
```

Comment:
The `method` and `class` values here don't seem necessary, since a link would be `GET` by default, and `usa-button--unstyled` is designed for styling a button to appear as a link, but this is a link already.

```suggestion
  <%= link_to t('mfa.skip'), @after_setup_path %>
```